common/src/Microsoft.Azure.IIoT.Hub.Processor/src/Services/EventProcessorFactory.cs

```diff
@@ -50,9 +50,15 @@ private class DefaultProcessor : IEventProcessor {
             /// Create processor
             /// </summary>
             /// <param name="outer"></param>
+            /// <param name="partitionContext"></param>
             /// <param name="logger"></param>
-            public DefaultProcessor(EventProcessorFactory outer, ILogger logger) {
+            public DefaultProcessor(
+                EventProcessorFactory outer,
+                PartitionContext partitionContext,
+                ILogger logger
+            ) {
```

Comment:
code style consistency?

Reply:
Fine if I change all methods here so that each parameter is on new line ?

Reply:
I don't mind the style as long is consistent through the project :)

Reply:
Fixed.